PsychSourceGL/Source/Common/Eyelink/EyelinkSynopsis.c

```diff
@@ -54,7 +54,7 @@ void InitializeSynopsis()
 	synopsis[i++] = "Eyelink('Shutdown')";
 	synopsis[i++] = "oldlevel = Eyelink('Verbosity' [,level]);";
 	synopsis[i++] = "Eyelink('TestSuite')";
-	synopsis[i++] = "[status =] Eyelink('OpenFile','filename')";
+	synopsis[i++] = "[status =] Eyelink('OpenFile', filename [, dontOpenExisting=0])";
```

Comment:
Hmm, so what you are saying is that this whole `dontOpenExisting` was broken from the beginning? And that user who requested the feature just never bothered to actually give me the feedback that `dontOpenExisting` doesn't work as intended? Users are the natural enemy of software developers :/

`open_data_file()` creates the EDF file on the tracker host, but `file_exists()` checks for it on the local display PC, which obviously makes non sense for the purpose of `dontOpenExisting`? The Eyelink programmers manual is not very clear about what function operates where, but my assumption and the one of the bug reporter was that it "obviously" operates on the tracker host.

If this is true then we need a different solution, which is sort of backwards compatible but doesn't surprise users in a bad way.

Could the function ``int create_path (char ∗ path, INT16 create, INT16 is_dir)`` be used as a proper replacement for `file_exists()`? Ie. does it operate on files on the tracker host computer? Can it just be used to query file existence.

E.g., would this work in the if statement for file_exists():

```
  if (0 == create_path(filename, 0, 0) {
    // Failure - file already exists...
  }
```
The text in the programmers manual suggests that calling it this way would **not ever** create a new file or directory, but just check and report if the file filename already exist, returning 0 if it already exists, and -1 otherwise?

If there is no way to do this properly, ie. checking for file existing on the tracker host computer, we should update the help text to explain that 'dontOpenExisting' exists for backwards compatibility, but is and always was actually unfixable broken. And then the best course of action would probably be to error out with a helpful error message whenever `dontOpenExisting` is set to a non-zero value, iow. whenever the user script asks for something that can't be done and that may lead to silent unexpected data loss.

Reply:
Ok, I went ahead and just made this change. Hopefully this was correct.